DAL/pojos/routineEntity.js

```diff
@@ -0,0 +1,90 @@
+module.exports = class routine {
+    routineId;
+    userId;
+    tittle;
+    privacy;
+    description;
+    exerciseInfo;
+    timestamp;
+   
+    
+    constructor(
+        routineId = undefined,
+        userId = undefined,
```

Comment:
only routineId and timestamp could be undefined.
change this line for:
userId,